backend/models/videoPromptModel.js

```diff
@@ -37,6 +37,8 @@ const VideoPromptInputSchema = new mongoose.Schema(
         message: String,
         fullMessage: String,
         model: String,
+        eta: Number,
+        fps: Number,
```

Comment:
samplingSteps and cfgScale was used in the create of this in video service, but not defined here.